detail.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from datetime import datetime, timezone
+from catalog import models as cmod
+
+def fix_html(x):
+    x = x.replace("&","&amp;")
+    x = x.replace("<","&lt;")
+    x = x.replace(">","&gt;")
+    return x
+
+@view_function
+def process_request(request,prod:cmod.Product=None):
+    #utc_time = datetime.utcnow()
```

Comment:
remove unnecessary lines that were commented out

Reply:
I should definitely do that.